packages/wallets/ONTO/src/adapter.ts

```diff
@@ -0,0 +1,201 @@
+import {
+    Adapter,
+    BaseMessageSignerWalletAdapter,
+    EventEmitter,
+    scopePollingDetectionStrategy,
+    WalletAccountError,
+    WalletConnectionError,
+    WalletDisconnectedError,
+    WalletName,
+    WalletNotConnectedError,
+    WalletNotReadyError,
+    WalletPublicKeyError,
+    WalletReadyState,
+    WalletSignTransactionError,
+} from '@solana/wallet-adapter-base';
+import { PublicKey, Transaction } from '@solana/web3.js';
+
+interface ONTOWalletEvents {
+    connect(...args: unknown[]): unknown;
+    disconnect(...args: unknown[]): unknown;
+}
+
+interface ONTOWallet extends EventEmitter<ONTOWalletEvents> {
+    isONTO?: boolean;
```

Comment:
No need to use `isONTO` for detection if you're not using `window.solana` :)

Reply:
have update